src/dom.js

```diff
@@ -380,6 +392,10 @@ equalizeNode.impl = (node, root = true, inlineTextNodes = []) => {
     collapseTextAcrossInlineNodes();
   }
 
+  if (node && node.nodeType === node.ELEMENT_NODE && node.className) {
```

Comment:
No need to check for `node` here; the start of the function contains `assertNode` so if null or undefined where passed, an error would be thrown. I also believe testing for `node.nodeType === node.ELEMENT_NODE` is superfluous since we already test for `node.className`.